charts/media-common/templates/deployment.yaml

```diff
@@ -27,6 +27,8 @@ spec:
       securityContext:
       {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
```

Comment:
this will create an empty initContainers key even if not used. Preferred format is

```yaml
{{- with .Values.initContainers }}
initContainers:
  {{- include "media-common.initContainers" . | nindent 8 }}
{{- end}}

Reply:
Better than an if statement too!